static/js/seta.js

```diff
@@ -39,63 +40,143 @@ $(function() {
 
 
   function printTable(date) {
-    $.getJSON("/data/setadetails/", {date:date}).done(outputTable);
+    $.getJSON("/data/setadetails/", {date:date}).done(function (data) { getActiveJobs(data, date); });
   }
 
+  function getActiveJobs(details, date) {
+    $.getJSON("/data/jobtypes/").done(function (data) { outputTable(data, details, date); });
+  }
+
+  // Simple text replace of full names -> Group-Code format
   function printName(testname) {
-    retVal = testname.replace('mochitest', 'm');
-    retVal = retVal.replace('m-browser-chrome', 'bc');
-    retVal = retVal.replace('m-other', 'm-oth');
-    retVal = retVal.replace('crashtest-ipc', 'c-ipc');
-    retVal = retVal.replace('crashtest', 'c');
-    retVal = retVal.replace('jsreftest', 'j');
-    retVal = retVal.replace('reftest', 'r');
-    retVal = retVal.replace('xpcshell', 'x');
+    var retVal = testname.replace('M-browser-chrome-', 'M-bc');
+    retVal = retVal.replace('M-browser-chrome', 'M-bc');
```

Comment:
If you used the regular expression form of replace, you could do something like:
  retVal.replace(/M-browser-chrome[-]?/, 'M-bc')
which would save some of the repetition here.